packages/core/.storybook/preview.js

```diff
@@ -42,6 +42,26 @@ const decorators = [
 const parameters = {
   controls: { expanded: true, hideNoControlsWarning: true },
   layout: 'centered',
+  options: {
```

Comment:
Would it make more sense to leverage the [`order` property](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories) on `storySort` here? I feel it would be both easier on the eyes and to maintain